doc/api/events.md

```diff
@@ -574,6 +574,39 @@ to indicate an unlimited number of listeners.
 
 Returns a reference to the `EventEmitter`, so that calls can be chained.
 
+### emitter.rawListeners(eventName)
+<!-- YAML
+added: REPLACEME
+-->
+- `eventName` {any}
```

Comment:
Hmm, shouldn't this be `{string|symbol}`

Reply:
It matches the rest of the docs and it is at least partially correct. We don't validate the input so if they pass in a number, it'll get automatically converted to a string.

Either way, if we're changing this then it should be changed in all of the docs.

Reply:
I've filed https://github.com/nodejs/node/issues/17657 to track that. This again LGTM.